.github/pull_request_template.md

```diff
@@ -1,15 +1,28 @@
 # Description
 
 Please include a summary of the changes and the related issue, if it exists. 
-Also, include relevant motivation and context for this PR. List any
-dependencies that are required for this change.
+Also, include relevant motivation and context for this PR. List any dependencies that are required for this change.
 
-Fixes # (issue)
+- Fixes # (issue)
 
 ## Did you add the right label?
 
 Remember to add the right labels to this PR.
 
 ## How was this tested?
 
-Describe what kind of tests for this functionality have been added.
+Describe the types of tests that have been added for this functionality
+
+## Did you update the documentation?
+
+Remember to ask yourself if your PR introduced changes to the following documentation:
```

Comment:
```suggestion
Remember to ask yourself if your PR effects or requires changes to the following documentation:
```

Because you're going to _introduce changes_ to the documents :D